### PR TITLE
update output syntax

### PR DESCRIPTION
**Pull request by nicgrayson**
Update output to new style from deprecation warning: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#environment-files

### CODE REVIEW

**Comment by nicgrayson** [comment]
@haya14busa pinging to make sure you saw this.

File: entrypoint.sh [hunk 1]
```diff
@@ -107,7 +107,7 @@ fi
 
 if [ -z "${BUMP_LEVEL}" ]; then
   echo "PR with labels for bump not found. Do nothing."
-  echo "::set-output name=skip::true"
+  echo "skip=true" >> $GITHUB_OUTPUT
```
Comment (shogo82148) [inline entrypoint.sh]:
```suggestion
  echo "skip=true" >> "$GITHUB_OUTPUT"
```

File: entrypoint.sh [hunk 2]
```diff
@@ -143,11 +143,11 @@ if [ -z "${NEXT_VERSION}" ]; then
   echo "Cannot find next version."
   exit 1
 fi
-echo "::set-output name=current_version::${CURRENT_VERSION}"
-echo "::set-output name=next_version::${NEXT_VERSION}"
+echo "current_version=${CURRENT_VERSION}" >> $GITHUB_OUTPUT
+echo "next_version=${NEXT_VERSION}" >> $GITHUB_OUTPUT
```
Comment (shogo82148) [inline entrypoint.sh]:
```suggestion
echo "current_version=${CURRENT_VERSION}" >> "$GITHUB_OUTPUT"
echo "next_version=${NEXT_VERSION}" >> "$GITHUB_OUTPUT"
```

File: entrypoint.sh [hunk 3]
```diff
@@ -143,11 +143,11 @@ if [ -z "${NEXT_VERSION}" ]; then
...
 
 TAG_MESSAGE="${NEXT_VERSION}: PR #${PR_NUMBER} - ${PR_TITLE}"
-echo "::set-output name=message::${TAG_MESSAGE}"
+echo "message=${TAG_MESSAGE}" >> $GITHUB_OUTPUT
```
Comment (shogo82148) [inline entrypoint.sh]:
```suggestion
echo "message=${TAG_MESSAGE}" >> "$GITHUB_OUTPUT"
```

**Review by shogo82148 — CHANGES_REQUESTED** [review]
Can you fix the warning of shellcheck?

File: entrypoint.sh (hunk 3 above)
Comment (nicgrayson) [reply entrypoint.sh]: Updated, Thanks!

**Review by shogo82148 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**